### PR TITLE
added version no to footer for deployment

### PR DESCRIPTION
**Pull request by Gurdeep475**
## Issue that this pull request solves

Closes: #4 

## Proposed changes

Footer now shows the latest git commit hash

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

image

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
 import React from "react";
+import GitInfo from 'react-git-info/macro';
+const gitInfo = GitInfo();
 
 const Footer = () => {
   return (
     <footer className="page-footer font-small pt-4">
       <div className="footer-copyright text-center py-3">
         Copyright © 2020 Rezume
+        <br/>
```
Comment (praveenscience) [inline src/components/Footer/Footer.js:10]: Can we have the indentation properly? Looks good otherwise.

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: src/components/Footer/Footer.js (hunk 1 above)
Comment (Gurdeep475) [reply src/components/Footer/Footer.js]: ran the project through the config mentioned.

**Review by praveenscience — CHANGES_REQUESTED** [review]
LGTM 👍

**Review by praveenscience — APPROVED** [review]
LGTM 👍

**Pull request merged**